fv_be/app/models/characters.py

```diff
@@ -30,9 +31,13 @@ class Meta:
     notes = models.TextField(blank=True)
 
     # from fvcharacter:related_words
-    # TODO: add this as a foreign key to the dictionary entry model once it's created
-    # related_dictionary_entries = models.ForeignKey(
-    #     DictionaryEntry, on_delete=models.SET_NULL, blank=True, null=True, related_name="characters")
+    related_dictionary_entries = models.ManyToManyField(
```

Comment:
Found a bug with this implementation, do not merge for now

Reply:
After giving it a shot I believe that this implementation with the horizontal filter is simpler. Is there a specifc reason to specify a through model?

Reply:
It's one of those things that is helpful for the future-- it gives us an explicit db table name and helps decouple the database schema from django's internal implementation choices. Here's a link with a longer explanation: https://medium.com/@DoorDash/tips-for-building-high-quality-django-apps-at-scale-a5a25917b2b5#6eea

Reply:
Ah I see, ok I will try to get my through implementation running the same as the current one